src/graph/graph.people.ts

```diff
@@ -9,18 +9,58 @@ import { Contact, Person, User } from '@microsoft/microsoft-graph-types';
 import { IGraph } from '../IGraph';
 import { prepScopes } from '../utils/GraphHelpers';
 
+/**
+ * Person Type enum
+ *
+ * @export
+ * @enum {number}
+ */
+export enum PersonType {
+  /**
+   * Any type
+   */
+  Any = 0,
+
+  /**
+   * A Person such as User or Contact
+   */
+  // tslint:disable-next-line:no-bitwise
```

Comment:
It might be advantageous to disable these rules for the entire types files, so you don't have to keep repeating it.

Reply:
I think there is a good reason to have the rule enabled globally to avoid accidently using bitwise operations when not intended.